flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/DiscardingSink.java

```diff
@@ -31,5 +31,5 @@
 	private static final long serialVersionUID = 1L;
 
 	@Override
-	public void invoke(T value) {}
+	public void invoke(T value, Context context) throws Exception {}
```

Comment:
Build failed because of this change (this is `@Public` class). Either let's keep it as it was before or you would have to somehow fix the api compatibility check (I don't remember at the moment how to handle it).

Reply:
Oops! I didn't expect it breaking the compatibility. I'll keep it as before.

Reply:
Neither did I ;)